.travis.yml

```diff
@@ -1,17 +1,20 @@
 language: node_js
 node_js:
-- '6'
 - '8'
 - '9'
```

Comment:
Ideally do `6`, `8`, `10` as that's the currently supported nodes.

Reply:
Separate note -- out node code can and should be node6+ going forward as a future ticket (definitely not needed here ;) )